### PR TITLE
html compatible for script tags

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ fix(#83)

### CODE REVIEW

File: lib/parse-template.js [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
-        const serializedList = transform.applyTransforms(baseTemplate, childTemplate);
-        resolve(serializedList);
+    return (baseTemplate, childTemplate) => new Promise((resolve, reject) => {
+        try {
+            const transform = new Transform(handledTags, insertBeforePipeTags);
+            const serializedList = transform.applyTransforms(baseTemplate, childTemplate);
+            resolve(serializedList);
+        } catch (e) {
+            reject(e);
+        }
```
Comment (boopathi) [inline lib/parse-template.js]: Why not 

```js
return () => Promise.resolve(new Transform(a,b)).then(transform => transform.applyTransforms(c,d));
```

here ... `new Transform(a,b)` should not throw ... if it did, it's a library error and we should fix it ... if `applyTransforms` throws then it's a lib-user error, and he might choose to handle it at the end of the chain.

Comment (vigneshshanmugam) [inline lib/parse-template.js]: as you said Transforms should not throw any error and Its handled in the lib itself.. Just to be extra sure to catch errors happening while parsing `templates`, Which shouldn't happen in any case.

File: lib/serializer.js [hunk 2]
```diff
@@ -38,6 +38,19 @@ module.exports = class CustomSerializer extends Serializer {
     }
 
     /**
+     * Gets the serialized content
+     * @returns {String}
+     */
+    getHtmlContent() {
```
Comment (mo-gr) [inline lib/serializer.js]: is this a one-time-getter? that should be made very clear at least in the jsdoc

Comment (vigneshshanmugam) [inline lib/serializer.js]: @mo-gr Can you check now..added a better explanation

**Comment by mo-gr** [comment]
:+1:

**Pull request merged**